packages/patternfly-4/react-core/README.md

```diff
@@ -64,23 +64,25 @@ Note: All commands below assume you are on the root directory in this repository
 
 ### Install
 
-Run to install all the dependencies
+Run to install all the dependencies.
 
 ```sh
 yarn install && yarn build
 ```
 
 ### Running
 
-To start the site locally.
+To start the site locally and view the site:
```

Comment:
Maybe consider referring to these like "start the _development server_" and/or "launch the _workspace_" would be more intuitive? I wonder if people will assume the right thing when they read "site".